completion.go

```diff
@@ -8,43 +8,57 @@ import (
 )
 
 const (
-	completionCommandName = "generate-completion"
-	completionFlagName    = "generate-shell-completion"
-	completionFlag        = "--" + completionFlagName
+	completionCommandName = "completion"
+
+	// This flag is supposed to only be used by the completion script itself to generate completions on the fly.
+	completionFlag = "--generate-shell-completion"
 )
 
+type renderCompletion func(cmd *Command, appName string) (string, error)
+
 var (
 	//go:embed autocomplete
 	autoCompleteFS embed.FS
 
 	shellCompletions = map[string]renderCompletion{
-		"bash": getCompletion("autocomplete/bash_autocomplete"),
-		"ps":   getCompletion("autocomplete/powershell_autocomplete.ps1"),
-		"zsh":  getCompletion("autocomplete/zsh_autocomplete"),
-		"fish": func(c *Command) (string, error) {
+		"bash": func(c *Command, appName string) (string, error) {
+			b, err := autoCompleteFS.ReadFile("autocomplete/bash_autocomplete")
```

Comment:
i wonder if we can ignore the error since we are loading from embedFS. Saves us from writing unit tests as well.

Reply:
nice catch! I think it's not worth worrying about the error here

Reply:
I just decided to not additionaly wrap the errors. I think it's fine now.

Reply:
Yes but the code cov decreased, I've worked so much over the last month to get the code coverage from 80 something percent to 98.4% and now it decreases again.

Reply:
Oh, I didn't think about it. I'm sorry!

What command can I run locally to see coverage percentage?